extractor.go

```diff
@@ -161,6 +161,11 @@ func (te *Extractor) Extract(reader io.Reader) error {
 		if err != nil || rel == "." {
 			return errInvalidRootMultipleRoots
 		}
+		for _, e := range strings.Split(rel, "/") {
```

Comment:
So, the tricky part is, this is a _file_ path. We probably need an `fp.ToSlash` call around `rel`.